INFO: Starting training:
        Epochs:          5
        Batch size:      1
        Learning rate:   1e-05
        Training size:   17
        Validation size: 1
        Checkpoints:     True
        Device:          cpu
        Images scaling:  0.5
        Mixed Precision: False
Epoch 1/5:   0%|                                        | 0/17 [00:01<?, ?img/s]
Traceback (most recent call last):
  File "/home/mikhail/github/Pytorch-UNet/train.py", line 212, in <module>
    train_model(
  File "/home/mikhail/github/Pytorch-UNet/train.py", line 106, in train_model
    loss = criterion(masks_pred, true_masks)
  File "/home/mikhail/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/mikhail/.local/lib/python3.10/site-packages/torch/nn/modules/loss.py", line 1174, in forward
    return F.cross_entropy(input, target, weight=self.weight,
  File "/home/mikhail/.local/lib/python3.10/site-packages/torch/nn/functional.py", line 3026, in cross_entropy
    return torch._C._nn.cross_entropy_loss(input, target, weight, _Reduction.get_enum(reduction), ignore_index, label_smoothing)
IndexError: Target 2 is out of bounds.
Traceback (most recent call last):
  File "/home/mikhail/github/Pytorch-UNet/train.py", line 212, in <module>
    train_model(
  File "/home/mikhail/github/Pytorch-UNet/train.py", line 106, in train_model
    loss = criterion(masks_pred, true_masks)
  File "/home/mikhail/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/mikhail/.local/lib/python3.10/site-packages/torch/nn/modules/loss.py", line 1174, in forward
    return F.cross_entropy(input, target, weight=self.weight,
  File "/home/mikhail/.local/lib/python3.10/site-packages/torch/nn/functional.py", line 3026, in cross_entropy
    return torch._C._nn.cross_entropy_loss(input, target, weight, _Reduction.get_enum(reduction), ignore_index, label_smoothing)
IndexError: Target 2 is out of bounds.